### ISSUE TITLE
Support installing an explicit version

### ISSUE THREAD

**Issue by mit-mit** [dart-lang/setup-dart#3]
Tracks installing a specific version from a specific channel (e.g. 2.9 stable). 

One possible option here is to enable passing a `version` string that corresponds to the `/<version>/` segment in our [download urls](https://dart.dev/tools/sdk/archive#download-urls).

E.g. for these Urls:

https://storage.googleapis.com/dart-archive/channels/stable/release/2.7.2/sdk/dartsdk-windows-ia32-release.zip

```
channel: stable
version: 2.7.2
```

https://storage.googleapis.com/dart-archive/channels/stable/release/2.1.1/sdk/dartsdk-macos-x64-release.zip

```
channel: stable
version: 2.1.1
```

https://storage.googleapis.com/dart-archive/channels/beta/release/2.8.0-20.11.beta/sdk/dartsdk-linux-x64-release.zip

```
channel: beta
version: 2.8.0-20.11.beta
```

https://storage.googleapis.com/dart-archive/channels/dev/release/2.9.0-1.0.dev/sdk/dartsdk-linux-x64-release.zip

```
channel: dev
version: 2.9.0-1.0.dev
```

**Comment by mit-mit** [comment]
Fixed in https://github.com/dart-lang/setup-dart/pull/10

**Comment by mit-mit** [comment]
And now available in `setup-dart@v0.2`; closing

**Comment by mit-mit** [comment]
Thanks to @aaronlademann-wf for the contribution!
